packages/splitio-adapter/modules/adapter/adapter.ts

```diff
@@ -186,9 +187,16 @@ class SplitioAdapter implements SplitioAdapterInterface {
     adapterArgs: SplitioAdapterArgs,
     adapterEventHandlers: AdapterEventHandlers
   ): Promise<any> {
-    const { authorizationKey, user, options = {} } = adapterArgs;
+    const {
+      authorizationKey,
+      user,
+      options = {},
+      treatmentAttributes,
+    } = adapterArgs;
 
     adapterState.user = ensureUser(user);
+    adapterState.treatmentAttributes = treatmentAttributes;
```

Comment:
Should these treatments also be changable from `reconfigure`?

Reply:
Hm, good point. I'll get an update up probably tomorrow. Thanks for the quick review!

Reply:
done!